### DISCUSSION TITLE
EQ6 Feedback

### DISCUSSION THREAD

**Discussion by siwelwerd** [TeamBasedInquiryLearning/precalculus#78] [Show and tell]
Section: https://teambasedinquirylearning.github.io/precalculus/2024/EQ6.html
Author: @AbbyANoble  
Primary Reviewer: @tdegeorge

**Comment by tdegeorge** [comment]
Activity 1.6.3: Second "denominator" should have an s **(should be plural)** and the word "excluded" is missing the d at the end.

There are two equal signs in the problem in part a.

Can we make one of the answer choices in part b also include -2 as that is one of the solutions if we were to factor the numerator?

**Reply by AbbyANoble** [reply]
The zeros of the numerator are 2 and 4 unless I'm factoring wrong! So they are included.

**Reply by tdegeorge** [reply]
Nope you are right! I factored wrong! :-p

**Comment by tdegeorge** [comment]
Activity 1.6.7: Did you want to make parts a and c multiple choice? I'm okay with either short answer or multiple choice, but just thought I'd ask since the previous activities had both those parts as multiple choice.

**Reply by AbbyANoble** [reply]
I was not planning to there, thinking they may be getting more fluent and not need the choices to guide them as much any more.

**Comment by tdegeorge** [comment]
Activity 1.6.9: For part a, do we want to include other possible values like 0 and 9 since those make the numerator zero?

Extra comma in part b for choice E

@AbbyANoble Just noticed that you are not quite done yet! Sorry! Let me know when you finish and I can come back and check the rest! :)

**Reply by AbbyANoble** [reply]
Fixed the typos and added in the other possible values. 

I added the last example (1.6.10) with less scaffolding and then 1.6.11 as fluency. About to submit a pull request so you should be able to see these changes soon!

**Comment by tdegeorge** [comment]
Activity 1.6.3: I see a typo: Which value(s) should be exclude (**should be excluded) as possible solutions to the following rational equations? @AbbyANoble

**Comment by tdegeorge** [comment]
Finished! :)
